example/admission-webhook/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: prometheus-operator-admission-webhook
+    app.kubernetes.io/version: 0.54.0
+  name: prometheus-operator-admission-webhook
+  namespace: default
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: prometheus-operator-admission-webhook
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: prometheus-operator-admission-webhook
+      labels:
+        app.kubernetes.io/name: prometheus-operator-admission-webhook
+        app.kubernetes.io/version: 0.54.0
+    spec:
+      automountServiceAccountToken: false
+      containers:
+      - image: quay.io/prometheus-operator/prometheus-operator:v0.54.0
```

Comment:
shouldn't it be `quay.io/prometheus-operator/prometheus-admission-webhook`?

Reply:
yup, thanks